exercises/js-logic.js

```diff
@@ -38,6 +58,12 @@ Ex: If the string that is passed in is `hey there person` the return value shoul
 */
 const stringReverse = (str) => {
 	// answer here
+	const words = str.split(' ')
+	
+	for (let i = 1; i < words.length; i++) {
```

Comment:
computer like to start counting at 0, you may have passed the tests here because 'hey' isn't supposed to be flipped but your loop starts with the second word